core/jobs/batch_jobs/exp_model_validation_jobs_test.py

```diff
@@ -0,0 +1,195 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.batch_jobs.exp_model_validation_jobs."""
+
+from __future__ import annotations
+
+from core import feconf
+from core.constants import constants
+from core.domain import state_domain
+from core.jobs import job_test_utils
+from core.jobs.batch_jobs import exp_model_validation_jobs
+from core.jobs.types import job_run_result
+from core.platform import models
+
+(exp_models,) = models.Registry.import_models([models.NAMES.exploration])
+
+
+class GetNumberOfInvalidExpsJobTests(job_test_utils.JobTestBase):
+
+    JOB_CLASS = exp_model_validation_jobs.GetNumberOfInvalidExpJob
+
+    EXPLORATION_ID_1 = '1'
+    EXPLORATION_ID_2 = '2'
+
+    STATE_2 = state_domain.State.create_default_state(
+        feconf.DEFAULT_INIT_STATE_NAME, is_initial_state=True).to_dict()
+
+    def setUp(self):
+        super().setUp()
+
+        # This is an invalid model.
+        self.exp_1 = self.create_model(
+            exp_models.ExplorationModel,
+            id=self.EXPLORATION_ID_1,
+            title='title',
+            init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
+            category=feconf.DEFAULT_EXPLORATION_CATEGORY,
+            objective=feconf.DEFAULT_EXPLORATION_OBJECTIVE,
+            language_code=constants.DEFAULT_LANGUAGE_CODE,
+            tags=['Topic'],
+            blurb='blurb',
+            author_notes='author notes',
+            states_schema_version=feconf.CURRENT_STATE_SCHEMA_VERSION,
+            param_specs={},
+            param_changes=[],
+            auto_tts_enabled=feconf.DEFAULT_AUTO_TTS_ENABLED,
+            correctness_feedback_enabled=False,
+            states={
+                'Introduction': {
+                    'content': {
+                        'content_id': 'content',
+                        'html': '<p><oppia-noninteractive-link text-with-value="&amp;quot;&amp;quot;" url-with-value="&amp;quot;http://google.com&amp;quot;"></oppia-noninteractive-link></p>', # pylint: disable=line-too-long
```

Comment:
Split this into multiple lines.

Reply:
ditto elsewhere where you use `# pylint: disable=line-too-long`

Reply:
Done!